components/query_filter/DEPS

```diff
@@ -0,0 +1 @@
+include_rules = [ "+third_party/re2/src/re2/re2.h" ]
```

Comment:
please add newlines at the end of all files. I would recommend configuring it in whatever editor you are using

Reply:
Done